src/Discord/WebSockets/Events/ApplicationCommandCreate.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is a part of the DiscordPHP project.
+ *
+ * Copyright (c) 2015-present David Cole <david.cole1340@gmail.com>
+ *
+ * This file is subject to the MIT license that is bundled
+ * with this source code in the LICENSE.md file.
+ */
+
+namespace Discord\WebSockets\Events;
+
+use Discord\Helpers\Deferred;
+use Discord\Parts\Interactions\Command\Command;
+use Discord\WebSockets\Event;
+
+class ApplicationCommandCreate extends Event
+{
+    /**
+     * @inheritdoc
+     */
+    public function handle(Deferred &$deferred, $data): void
+    {
+        $deferred->resolve($this->factory->create(Command::class, $data, true));
```

Comment:
Does this need to be inserted into a repository (`GlobalCommandRepository` or `GuildCommandRepository`)?

Reply:
Oh I guess it's already inserted if we are creating one?

Reply:
It's already inserted if we are creating one, however this one was for the APPLICATION COMMAND gateway events that seems not mean for bot (i.e. I thought the bot can receive this event if the command was created else where), I'm removing it.